### ISSUE TITLE
Problem with check_table function

### ISSUE THREAD

**Issue by BrunoMine** [Gerold55/ma_pops_furniture#2]
![erro mod](https://user-images.githubusercontent.com/7986261/46577128-fcf4d480-c9b4-11e8-9ef3-6642319f463b.png)

**Comment by daytonnvpaul** [comment]
Not sure if this is a fix, but it was recommended by a fellow admin to add this after line 1 in init.lua.  Seemed to work for me.

ma_pops_furniture.check_table = function() end

**Comment by BrunoMine** [comment]
Exactly, this function does not exist, maybe a part of the code has not been developed yet?

**Comment by mrbid** [comment]
A bug discovered when calling the check_table() function causes a crash when certain tables of a non-expected naming convention are used, specifically that I am aware of, the wood and cobble tables. The solution I have opted for to comment out all calls to this function as I deem the optimization of legs when placing tables next to one another superfluous .. however this may change in the future as user "Test_User" put in some considerable time to find the exact problem as listed above and provide a solution to said problem as listed below:

functions.lua on line 114, should be able to replace if check_this then with if check_this and material ~= "_wood" then
or, if the wood one did have other forms, at the start if material == "_wood" then material = "wood" end
